src/_gettsim/transfers/arbeitsl_geld_2/bedarf.py

```diff
@@ -3,6 +3,73 @@
 
 from _gettsim.shared import policy_info
 
+aggregate_by_group_benefit_checks = {
+    "kinder_mit_gedecktem_bedarf_in_fg": {
+        "source_col": "arbeitsl_geld_2_eigenbedarf_gedeckt_und_kind_in_fg",
+        "aggr": "any",
+    },
+}
+
+
+def arbeitsl_geld_2_eigenbedarf_gedeckt(  # noqa: PLR0913
+    arbeitsl_geld_2_nettoeink_vor_abzug_freibetrag_m: float,
+    arbeitsl_geld_2_regelbedarf_m: float,
+    kindergeld_zur_bedarfsdeckung_m: float,
+    kind_unterh_erhalt_m: float,
+    unterhaltsvors_m: float,
+    kindergeldübertrag_m: float,
+) -> bool:
+    """Needs according to SGB II are covered.
+
+    Parameters
+    ----------
+    arbeitsl_geld_2_nettoeink_vor_abzug_freibetrag_m
+        See :func:`arbeitsl_geld_2_nettoeink_vor_abzug_freibetrag_m`.
+    arbeitsl_geld_2_regelbedarf_m
+        See :func:`arbeitsl_geld_2_regelbedarf_m`.
+    kindergeld_zur_bedarfsdeckung_m
+        See :func:`kindergeld_zur_bedarfsdeckung_m`.
+    kind_unterh_erhalt_m
+        See :func:`kind_unterh_erhalt_m`.
+    unterhaltsvors_m
+        See :func:`unterhaltsvors_m`.
+    kindergeldübertrag_m
+        See :func:`kindergeldübertrag_m`.
+
+    Returns
+    -------
+
+    """
+    return (
+        arbeitsl_geld_2_nettoeink_vor_abzug_freibetrag_m
+        + kindergeld_zur_bedarfsdeckung_m
+        + kind_unterh_erhalt_m
+        + unterhaltsvors_m
+        + kindergeldübertrag_m
+        >= arbeitsl_geld_2_regelbedarf_m
+    )
+
+
+def arbeitsl_geld_2_eigenbedarf_gedeckt_und_kind_in_fg(
```

Comment:
Why can't we just include this in the above function? I.e., do we ever need the `arbeitsl_geld_2_eigenbedarf_gedeckt` without the `ist_kind_in_fg` condition?

AFAICT:
- We are using the joint condition for `kinder_mit_gedecktem_bedarf_in_fg` and in `wohngeld_kinderzuschl_statt_arbeitsl_geld_2`
- We are replicating the joint condition *exactly* in `wthh_id_numpy` and `wohngeld_kinderzuschl_günstiger`
- We are implicitly replicating the joint condition in `bg_id_numpy` (there even is an issue to make it explicit)